presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -70,7 +72,7 @@ public void testAcos()
     {
         for (double doubleValue : DOUBLE_VALUES) {
             assertFunction("acos(" + doubleValue + ")", DOUBLE, Math.acos(doubleValue));
-            assertFunction("acos(FLOAT '" + (float) doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
+            assertFunction("acos(FLOAT '" + doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
```

Comment:
Okay there were other places where it was just taking doubleValue in existing tests that you added so I wasn't sure what your intent was. Lets chat and decide which approach is better.